src/sdk/utils/Assets.ts

```diff
@@ -0,0 +1,41 @@
+export class Assets {
+  static assetCount = 0;
+  static loadingAssetUrls = [];
+  static onProgressFns: ((loaded: number, total: number) => void)[] = [];
+  static onLoadFns: (() => void)[] = [];
+
+  /**
+   * Returns the appropriate URL for an asset and also schedules it for preloading.
+   */
+  static getAssetUrl(asset: string) {
+    const url = `https://assets-soltrainer.netlify.app/${asset}`;
```

Comment:
:eyes: should this be pointed somewhere else at some point?

Reply:
Not yet. Only I have the assets, let's get the IP issue sorted first. It will work for you, the CORS origin is set to `*`